### PR TITLE
fix: remove scroll when drawing on mobile

### PR DESCRIPTION
**Pull request by Taloud**
Petite pierre a l'édifice pour faciliter sur le mobile :)

### CODE REVIEW

**Comment by magopian** [comment]
Merci beaucoup @Taloud ! Je n'avais pas de soucis sur android, peut-être que Safari ne supporte pas `touch-action: none`.

File: index.html [hunk 1]
```diff
@@ -489,6 +489,7 @@ <h2>
       }
 
       function touchstartEventHandler(e) {
+        event.preventDefault();
```
Comment (davidbgk) [inline index.html:492]: I think it should be `e.preventDefault()` in that case.

File: index.html [hunk 2]
```diff
@@ -515,6 +517,7 @@ <h2>
       }
 
       function touchMoveEventHandler(e) {
+        event.preventDefault();
```
Comment (davidbgk) [inline index.html:520]: Same here.

**Review by davidbgk — COMMENTED** [review]
Maybe it would be better to uniformize the way events are called ;)

**Comment by magopian** [comment]
Ah, I merged too quickly. Weird that it still worked?! I'm pushing a fix, thanks a lot @davidbgk for the heads up ;)

**Comment by magopian** [comment]
Fixed in https://github.com/magopian/att/commit/b8c402b0a84dbb163153208ca79416b92e86ac55

**Pull request merged**